src/xdisplay.cpp

```diff
@@ -13,61 +13,18 @@
 
 #include "xeus/xinterpreter.hpp"
 
-#include "pybind11/pybind11.h"
 #include "pybind11/embed.h"
+#include "pybind11/pybind11.h"
 
-#include "xutils.hpp"
 #include "xdisplay.hpp"
+#include "xutils.hpp"
 
 namespace py = pybind11;
 namespace nl = nlohmann;
 
 namespace xpyt
 {
-    xdisplayhook::xdisplayhook() : m_execution_count(0)
```

Comment:
called it `mime_bundle_repr` so that it may as well be the standard `mime_bundle_repr` in xeus-cling for `pybind11::object`.